tasks/main.yml

```diff
@@ -29,3 +29,25 @@
     name: "{{ rabbitmq_daemon }}"
     state: "{{ rabbitmq_state }}"
     enabled: "{{ rabbitmq_enabled }}"
+
+- name: Ensure rabbitmq vhosts
+  rabbitmq_vhost:
+    name: "{{ item }}"
+  with_items: "{{ rabbitmq_vhosts }}"
+
+- name: Ensure the users present
+  rabbitmq_user:
```

Comment:
Hello!
 I think a few edits are needed here:

`community.rabbitmq.rabbitmq_user`

And the same for vhost

Reply:
After fixes i will approved this changes. Please make request for ours github actions ci tests.

Reply:
pushed commit using FQCN